Sources/Segment/ObjC/ObjCAnalytics.swift

```diff
@@ -13,12 +13,20 @@ import Foundation
 
 @objc(SEGAnalytics)
 public class ObjCAnalytics: NSObject {
-    internal let analytics: Analytics
+    /// The underlying Analytics object we're working with
+    public let analytics: Analytics
```

Comment:
We had to expose this for tests?

Reply:
makes cross-config way easier if you can init in objc and then access it later in swift if needed and vice versa.